reference/mongodb/bson/serializable.xml

```diff
@@ -30,20 +30,46 @@
      <ooclass>
       <classname>MongoDB\BSON\Serializable</classname>
      </ooclass>
-     
+
      <oointerface>
       <interfacename>MongoDB\BSON\Type</interfacename>
      </oointerface>
     </classsynopsisinfo>
 <!-- }}} -->
-    
+
     <classsynopsisinfo role="comment">&Methods;</classsynopsisinfo>
     <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.mongodb-bson-serializable')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[not(@role='procedural')])" />
    </classsynopsis>
 <!-- }}} -->
 
   </section>
 
+  <section role="changelog">
+   &reftitle.changelog;
+   <para>
+    <informaltable>
+     <tgroup cols="2">
+      <thead>
+       <row>
+        <entry>&Version;</entry>
+        <entry>&Description;</entry>
+       </row>
+      </thead>
+      <tbody>
+       <row>
+        <entry>PECL mongodb 1.15.0</entry>
+        <entry>
+         Return types for methods are declared as tentative on PHP 8.0 and newer,
+         triggering deprecation notices in code that implements this interface
+         without declaring the appropriate return types. The <code>#[ReturnTypeWillChange]</code>
```

Comment:
`en/reference/mongodb/architecture.xml` and `en/reference/mongodb/bson/serializable/bsonserialize.xml` both have example implementations of this interface, where you might want to add `#[ReturnTypeWillChange]` attributes.